sympy/combinatorics/fp_groups.py

```diff
@@ -17,7 +17,9 @@
 
 
 @public
-def fp_group(fr_grp, relators=[]):
+def fp_group(fr_grp, relators=None):
+	if relators is None:
+		relators = []
```

Comment:
```suggestion
def fp_group(fr_grp, relators=()):
	  relators = list(relators)
```
which avoids mutating the argument when it is explicitly provided too.

Reply:
This is a previous commit actually. I force pushed to it as there were a lots of issues with my PR including merging conflicts. I'll soon make the changes according to what you suggested.